### DISCUSSION TITLE
Estilização do PO-MODAL

### DISCUSSION THREAD

**Discussion by daancorrea** [po-ui/po-angular#1353] [Tire suas dúvidas] [answered]
Olá, 

No projeto tenho que ter um modal para "Filtros" que se abre no lado direito da tela, ocupando 100% da altura da tela e a largura em torno de 500px. 
Tem alguma propriedade que pode ser passada para estilizar o modal, pra continuar com as funcionalidades de abrir e fechar? Ou preciso fazer um modal como eu quero na mão?

**Comment by daancorrea** [comment]
Fuçando o projeto, consegue fazer todas as alterações que eu gostaria pelo "src/styles.css" ou seja, mexendo nos estilos globais da minha aplicação, pegando a classe ".po-modal-overlay" para alterar a cor de fundo por exemplo. 
Minha dúvida é se alterar os estilos padrões do PO-UI usando esse método, é considerado uma má pratica ou vocês recomendam que faça isso caso o estilo seja diferente do padrão?

**Comment by deboraconstantino** [comment, marked as answer]
O componente [page slide](https://po-ui.io/documentation/po-page-slide) não atende a sua necessidade? Ele abre no lado direito da tela, ocupando 100% da altura
